datageneration/prompts/personas.txt

```diff
@@ -1,7 +1,26 @@
-political journalist
-investigative journalist
-expert fact checker
-hobby fact checker
-human rights abuse monitoring OSINT Expert
-OSINT beginner
-legal professional
\ No newline at end of file
+local resident exploring neighborhood
+tourist visiting for the first time
+parent planning a day with kids
+teenager looking for a hangout spot
+someone looking for a place late at night
+non-native speaker German
+non-native speaker Spanish
+non-native speaker Chinese
+non-native speaker Balto-Slavic Language
+ESL speaker with basic vocabulary
+visually impaired user using voice
+elderly person not tech-savvy
+GIS analyst exploring map data
+college student using a map app
+urban planner doing neighborhood checks
+real estate agent checking surroundings
+delivery driver planning a route
+ride-share driver finding landmarks
+field reporter investigating a location
+travel blogger writing location notes
+AI assistant trying to sound human
+a time traveler visiting the modern day
+conspiracy theorist mapping patterns
```

Comment:
i would remove this. Checked the example of generated sentences they are mostly labeled as unrealistic combinations
or why/what type questions:
-why is there a fish restaurant just 38,42 in from a protected area and how does an arts & crafts store fit into this being around 207,00 m away
-Why would an instrument shop and a radar speed sign both be located inside a ditch in Santa Fe?